contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -63,9 +63,9 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
 
     // minimum duration in seconds for a campaign
     uint256 public minCampaignDurationInSeconds = 345600; // around 4 days
-    IERC20 public kncToken;
+    IERC20 public immutable kncToken;
     IKyberStaking public staking;
```

Comment:
staking + minCampaignDurationInSeconds as well I think

Reply:
Ah, can't do for staking cos its methods are read in the constructor...
Unless we do `_staking.epochPeriodInSeconds()` instead of `staking.epochPeriodInSeconds()`

Reply:
As for `minCampaignDurationInSeconds`, it's being overwritten by mocks....